docs/data/pages.ts

```diff
@@ -93,6 +93,9 @@ const pages: MuiPage[] = [
           { pathname: '/x/react-data-grid/state' },
         ],
       },
+      {
+        pathname: '/x/react-data-grid/recipes',
```

Comment:
We can do multiple pages as well (similar to `Advanced` category in the nav bar), but:
- those pages would be rather short
- it's easier to add new demo to existing page than create a new one

Reply:
Did you find any other example not related to editing? My concern is if we won't have a page that most recipes are related to editing. We already have https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/editing/#advanced-use-cases which are recipes. Maybe the sidebar should be:

Editing
- Fundamentals (goes to https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/editing/)
- Recipes (https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/editing/#advanced-use-cases + https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/recipes/)

Or we move everything from https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/editing/#advanced-use-cases to https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/recipes/

Reply:
> Did you find any other example not related to editing?

A lot of them are indeed related to editing - like https://github.com/mui/mui-x/issues/5381
But we will probably have some not related to editing as well - like https://github.com/mui/mui-x/issues/4127 or https://github.com/mui/mui-x/issues/3656

+1 for moving "Advanced use cases" examples from "Editing" to "Recipes".